geometry_msgs/Point target
float64 target_distance
float64 target_angle
dwa_tutorial/Obstacle[] obstacles